### DISCUSSION TITLE
show the gp required to turn herbs into unfinished potions on the Banked Xp plugin

### DISCUSSION THREAD

**Discussion by Evenfisher1** [runelite/runelite#16410] [Ideas]
this is already done for banked construction xp, showing how much gp would be needed to turn your logs into planks. the Plugin already shows the vials of water needed as a secondary so that number would have to be multiplied by 200 to get the amount of gp needed.

**Comment by Nightfirecat** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.
